pkg/controller/manager/sync_action.go

```diff
@@ -6,10 +6,11 @@ import (
 	"context"
 	"fmt"
 	"reflect"
+	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/scylladb/go-set/strset"
-	"github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	v1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
```

Comment:
nit: do the rename commit first so you don't have to modify the same line in one PR

Reply:
done